data/templates/default.docbook5

```diff
@@ -6,8 +6,12 @@ $if(mathml)$
   xmlns:mml="http://www.w3.org/1998/Math/MathML"
 $endif$
   xmlns:xlink="http://www.w3.org/1999/xlink" >
+$if(lang)$ 
+      xml:lang="$lang$"
+$endif$ >
   <info>
     <title>$title$</title>
+    <subtitle>$subtitle$</subtitle>
```

Comment:
Perhaps this should be made conditional, so we don't get an empty `<subtitle>` element if there's no subtitle?